content/en/04-tutorials/01-pixie-101/07-kafka-monitoring.md

```diff
@@ -0,0 +1,184 @@
+
+```bash
+kubectl -n px-kafka get svc apache
+```
+
+To use the application, click on the Order page and scroll to the bottom of the page to select the `"Add Order"` button. Any orders added by the order service should soon show up on both the Shipping and Invoicing pages.
+
+## Topic Flow Graph
+
+Let's use Pixie to see a graph of producers and consumers for each Kafka topic.
+
+1. Select `px/kafka_overview` from the script drop-down menu.
+
+::: div image-xl relative
+<svg title='' src='use-case-tutorials/kafka/kafka-overview.png'/>
+:::
+
+> This graph shows us that we have 1 topic, named `order`, with 1 producer and 2 consumers.
+
+<Alert variant="outlined" severity="info">If this script doesn't show any output, make sure that you started the load generator in Prerequisites Step #3.</Alert>
+
+2. Hover over an edge on the graph to see throughput and total record Bytes for a producer or consumer. The thickness of the edge indicates an increase in throughput.
+
+> By examining the edges of the graph, we can see that the producer is producing to the order topic at about 450 B/s. The `shipping` consumer is consuming at the same rate of 450 B/s. The `invoicing` consumer is only consuming at 86 B/s and is falling behind.
+
+> The table at the bottom summarizes the same topic information, but also includes the number of topic partitions.
+
+## Latency and Throughput per Pod
+
+The previous script alerted us to the fact that the `invoicing` pod is consuming messages at a slower speed than they are being produced. Let's investigate this more.
+
+1. Select `px/kafka_stats` from the script drop-down menu.
+
+> This script calculates the latency and throughput of a pod's Kafka requests.
+
+::: div image-xl relative
+<svg title='' src='use-case-tutorials/kafka/kafka-stats.png'/>
+:::
+
+> The **Summary** table at the bottom of the page shows us the request throughput, latency, and total number of Kafka requests per pod. The `shipping` pod has sent around 7,600 `Fetch` requests while the `invoicing` pod has sent much fewer (only 1700 `Fetch` requests). We can also see that the This again indicates that something is wrong.
+
+2. To check the pod's resources, click on the `px-kafka/invoicing-*` pod name in the **SOURCE** column of the **Summary** table to follow the deep link to the `px/pod` script.
+
+> This script shows an overview of the pod's CPU usage, network traffic and throughput, disk and memory usage.
+
+> In this case, CPU utilization is low, which means resource starvation is not causing the `invoicing` pod to consume messages at a slower rate.
+
+## Full-body Kafka Requests
+
+Let's inspect the raw Kafka requests flowing through the cluster.
+
+1. Select `px/kafka_data` from the script drop-down menu.
+
+> This script shows a sample of the most recent Kafka messages in your cluster.
+
+> For each record you can see the source, destination, request command, full-body request and response, and the latency.
```

Comment:
Maybe a note saying that: We only show size of the payload, not the content, because it's usually too large.